web/modules/custom/dbcdk_community/src/Content/FlaggableContentRepository.php

```diff
@@ -100,20 +101,13 @@ protected function getContent(array $filter = []) {
       // We do not know which type of content our flag is attached to and
       // in theory it could be attached to multiple. We get all of them and
       // convert them to FlaggableContent instances.
-      try {
-        $posts = (array) $this->flagApi->flagPrototypeGetPosts($flag->getId());
-        $comments = (array) $this->flagApi->flagPrototypeGetComments($flag->getId());
-      }
-      catch (ApiException $e) {
```

Comment:
@kasperg it doesn't look like ApiException is being used anymore. Should we remove it as a "use" and declare the fully qualified namespace for `@throws` in the getContentWithUnreadFlags() DocBlock?